File: app/src/release/java/com/leaguechampions/utils/UrlUtils.java
package com.leaguechampions.utils;

import android.content.Context;

import com.leaguechampions.data.local.Const;
import com.leaguechampions.data.model.Champion;

public final class UrlUtils {

    private UrlUtils() {}

    public static String getImageUrl(Context context, Champion champion) {
        return Const.URL_BASE + "cdn/" + champion.getVersion() + "/img/champion/" + champion.getId() + ".png";
    }
}

File: app/src/release/java/com/leaguechampions/utils/UrlUtils.kt
package com.leaguechampions.utils

import android.content.Context
import com.leaguechampions.data.local.Const
import com.leaguechampions.data.model.Champion

object UrlUtils {

    fun getImageUrl(context: Context, champion: Champion): String {
        return "${Const.URL_BASE}cdn/${champion.version}/img/champion/${champion.id}.png"
    }
}
